### PR TITLE
Add Python 3.10 to the build matrix

### PR DESCRIPTION
**Pull request by bgruening**
The idea is to test the lower and upper bound.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7']
+        python-version: ['3.7', '3.10']
```
Comment (mvdbeek) [inline .github/workflows/ci.yaml:20]: That double the long running test time. can we skip the 3.10 tests on everything but `unit-quick` ?

**Comment by mvdbeek** [comment]
Just a note, I've disabled new branch creation to avoid all the extra CI here.
Screenshot 2023-06-21 at 12 11 22

**Comment by nsoranzo** [comment]
> Just a note, I've disabled new branch creation to avoid all the extra CI here. 

Thanks a lot, that works much better than reminding people! I can close and reopen from my fork.

**Comment by nsoranzo** [comment]
> Just a note, I've disabled new branch creation to avoid all the extra CI here.

Thanks for the screenshot, I've done the same for the BioBlend repo!

**Pull request closed without merging**